docs/documentation/aurora-config.md

```diff
@@ -648,6 +648,25 @@ logging:
   index: ""
 ```
 
+### Topology
+Support for organizing the topology view is available with the `topology` configuration.
+
+| Name                  | Default | Description                                                             |
+|-----------------------|---------|-------------------------------------------------------------------------|
+| `topology/partOf`     |         | Define the group the deployment belongs to.                             |
+| `topology/runtime`    |         | Single value defining the runtime (i.e spring-boot) of the application. |
+| `topology/connectsTo` |         | List of application names the deployment connects to.                   |
```

Comment:
Hva skal brukeren putte her? Name hvis det er satt fra auroraconfig, hvis ikke filnavnet på applikasjonen. Kan godt være jeg pirker litt mye på denne altså og at det er klart i seg selv

Reply:
Jeg la til litt mer info i notes seksjon under, men vi bør kanskje ha en terminologi seksjon for application name og lignende på starten av dokumentasjonen?